File: lecture_001/oddeven.java
import java.util.Scanner;
//verified
public class oddeven{
    public static void main(String[] args){
        Scanner scn=new Scanner(System.in);
        int a=scn.nextInt();
        if(a % 2 !=0){
            System.out.print(a + " is odd");
        }
        else{
            System.out.print(a + " is even");
        }
    }
}

File: lecture_001/oddeven.cpp
#include<iostream>
//verified
using namespace std;
int main(int argc, char**argv){
	int a;
	cout<<"enter no. : ";
	cin>>a;
	if(a % 2){
		cout<<a<<" is odd";
	}
	else{
		cout<<a<<" is even";
	}
	return 0;
}